src/main/scala/org/allenai/deep_qa/pipeline/BackgroundCorpusSearcherStep.scala

```diff
@@ -97,14 +95,39 @@ class DefaultBackgroundCorpusSearcherStep(
   override def _runStep() {
     // TODO(matt): might want to make this streaming, for large input files
     val lines = fileUtil.readLinesFromFile(sentencesFile)
-    val indexedSentences = lines.map(line => {
+    val indexedSentences: Seq[(Int, Seq[Query])] = lines.map(line => {
       val fields = line.split("\t")
       sentenceFormat match {
-        case "plain sentence" => (fields(0).toInt, Seq(fields(1)))
+        case "plain sentence" => (fields(0).toInt, Seq(new StringQuery(fields(1))))
         case "question and answer" => {
+          val qText = fields(1)
           val answers = fields(2).split("###")
-          val queries = answers.map(fields(1) + " " + _).toSeq
-          (fields(0).toInt, Seq(fields(1)) ++ queries)
+          val stringQueries = answers
+              .map(a => new StringQuery(qText + " " + a))
```

Comment:
This is on its own line, so there's plenty of horizontal space.  I would avoid the abbreviation here, using `answer` instead of `a`.